malariagen_data/anoph/cnv_data.py

```diff
@@ -399,7 +413,13 @@ def open_cnv_discordant_read_calls(
         except KeyError:
             release = self.lookup_release(sample_set=sample_set)
             release_path = self._release_to_path(release)
-            path = f"{self._base_path}/{release_path}/cnv/{sample_set}/discordant_read_calls/zarr"
+            analysis = self._discordant_read_calls_analysis
+            if analysis:
+                calls_version = f"discordant_read_calls_{analysis}"
+            else:
+                calls_version = "discordant_read_calls"
```

Comment:
If we use the empty string `""` to indicate the original analysis version, then this should work as desired. I.e., the empty string will evaluate to False.